components/RoadmapForm.tsx

```diff
@@ -103,31 +105,33 @@ export function RoadmapForm() {
   });
 
   return (
-    <form onSubmit={formSubmit}>
-      <FormControl isInvalid={error != null} isDisabled={globalLoadingState.get()}>
-        <InputGroup>
-          <InputLeftElement
-            pointerEvents='none'
-            children={<SearchIcon color='#FFFFFF' />}
-          />
-          <Input
-            type="text"
-            value={currentIssueUrl}
-            className={styles.urlInput}
-            color={theme.light.header.input.text.color}
-            aria-label='Issue URL'
-            name='issue-url'
-            autoComplete='url'
-            onChange={onChangeHandler}
-            placeholder='https://github.com/...'
-            bg={theme.light.header.input.background.color}
-            borderColor={theme.light.header.input.border.color}
-            borderRadius={theme.light.header.input.border.radius}
-          />
-          <InputRightElement cursor="pointer" children={inputRightElement}/>
-        </InputGroup>
-        <FormErrorMessage>{error?.message}</FormErrorMessage>
-      </FormControl>
-    </form>
+    isSmallScreen ?
+      <SearchIcon color='#FFFFFF' /> :
```

Comment:
I would much prefer an escape hatch on `isSmallScreen` rather than an entire shifting of the existing return. It's very difficult to tell if you changed anything in the existing code.